end-to-end-tests/specs/screenshot.spec.js

```diff
@@ -93,10 +93,89 @@ describe('result page tabs', function(){
         var res = browser.checkElement('#bookmark_email', {hide:['.qtip'] });
         ssAssert(res);
     });
+});
+
+describe('result page tabs, loading from session id', function(){
+    before(function(){
```

Comment:
these tests are duplicates of the ones above, except starting from URL with session_id